.github/PULL_REQUEST_TEMPLATE/rfc.md

```diff
@@ -1,9 +1,8 @@
 ---
-name: RFC Template
-about: A design RFC for Zebra
-title: ''
-labels: C-design
-assignees: ''
+Title: RFC Template
+About: A design RFC for Zebra
+Labels: C-design
+Assignees: ''
```

Comment:
I'm not sure if we can change these fields, they are already in the exact format required by GitHub

Reply:
I see, thanks, changed that back.

Reply:
What format github requires exactly and what is the purpose of making it that way instead of coding a normal markup table with the fields we want ?

I am asking it because when you add the header into a github issue the table as it is coded now is not even visible.
(https://github.com/ZcashFoundation/zebra/pull/1129)

Reply:
The GitHub header automatically fills in the name and labels in the ticket metadata. So the table doesn't need to be visible, because it would just duplicate that information.

For more details, see:
https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/manually-creating-a-single-issue-template-for-your-repository